CHARTER.md

```diff
@@ -0,0 +1,63 @@
+
+The scope of project includes documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
+
+### 1.1: In-scope
+
+ - Overseeing the **technical development** of the core vis.gl libraries and all affiliated sub-projects.
+ - **Release planning** and coordination
+ - Managing project repositories and **governance**, including issue tracking, pull request processes, and repository organization.
+ - Defining and enforcing **coding standards, testing practices, and development processes** to maintain code quality.
+ - Maintaining the list of **Maintainers and core contributors**, and setting permissions in project repositories.
+ - Reviewing and proposing **new project incubations or sub-projects**, as well as deciding on deprecations or significant changes to the project’s scope.
+ - Formation of technical **working groups**.
+ - **Representation** in technical collaborations with external communities.
+
+### 1.2: Out-of-Scope
+
+Section Intentionally Left Blank
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+Technical leadership for the projects within the [OpenJS Foundation](https://openjsf.org/) is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the `vis.gl` project, it is delegated to the [Steering Committee Members](https://github.com/visgl/tsc/blob/master/README.md#steering-committee-members) (the “TSC”). The OpenJS Foundation's business leadership is the Board of Directors (the “Board”).
+
+This `vis.gl` Charter reflects a carefully constructed balanced role for the TSC and the CPC in the governance of the OpenJS Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the CPC. The CPC may additionally make amendments to the project charter at any time, though the CPC will not interfere with day-to-day discussions, votes, or meetings of the TSC.
+
+### 2.1 Other Formal Project Relationships
+
+Section Intentionally Left Blank
+
+## Section 3: `vis.gl`'s TSC Governing Body
+
+`vis.gl` is governed by its [Steering Committee Members](https://github.com/visgl/tsc/blob/master/README.md#steering-committee-members).
+
+## Section 4: Roles & Responsibilities 
+
+The roles and responsibilities of `vis.gl`'s TSC are described in [GOVERNANCE.md](./GOVERNANCE.md).
+
+### Section 4.1 Project Operations & Management
+
+Section Intentionally Left Blank
+
+### Section 4.2: Decision-making, Voting, and/or Elections
+
+The project will define a consensus-seeking governance model in [GOVERNANCE.md](./GOVERNANCE.md) as need arises.
```

Comment:
I'd rather this used the CPC's one instead of punting to later.

Reply:
I don't think we're punting this - we have a decision making process defined in the governance.

Could you clarify what you're recommending? I had pulled this from Jest's charter (also used in NVM and some others). I'm curious when we'd want to not align with the other projects?

Reply:
It's added to the charter